source/get-started/bootable-usb.rst

```diff
@@ -22,90 +23,140 @@ Burn the |CL| image onto a USB drive
    Burning an image formats the USB drive and destroys all pre-existing
    content.  Back up your data before proceeding.
 
-#. Launch balenaEtcher.
+#. Launch Etcher.
 
    .. figure:: /_figures/bootable-usb/balenaEtcher_Start.PNG
       :scale: 100%
-      :alt: BalenaEtcher Start
+      :alt: Start screen
          
-      Figure X: BalenaEtcher Start
+      Figure 1: Start screen
  
 #. Press :guilabel:`Select Image`.
 
 #. Change directory to where the image resides.
 
-#. Open the Image.
+#. Select the image and click :guilabel:`Open`.
 
-   .. figure:: /_figures/bootable-usb/balenaEtcher_ImgaeSlect.PNG
+   .. figure:: /_figures/bootable-usb/balenaEtcher_ImageSelect.PNG
       :scale: 100%
-      :alt: 
+      :alt: In Open, select the image
          
-      Figure X: 
+      Figure 2: In Open, select the image
 
 #. Plug in the USB drive.
 
-#. Identify the USB drive using the :guilabel:`Change` under the middel icon.
+#. Identify the USB drive or click :guilabel:`Change` to select a 
+   different USB.
     
    .. note::
+
       This shows all USB drives attached to the system.
 
    .. figure:: /_figures/bootable-usb/balenaEtcher_DriveSlect.PNG
       :scale: 100%
-      :alt: 
+      :alt: USB drives attached
          
-      Figure X: 
+      Figure 3: USB drives attached
 
-#. Select the proper drive and press :guilabel:`Continue`.
+#. Select the proper device and press :guilabel:`Continue`.
 
    .. figure:: /_figures/bootable-usb/balenaEtcher_ReadyToFlash.PNG
       :scale: 100%
-      :alt: 
+      :alt: USB Flash Device selected
       
-      Figure X: 
+      Figure 4: USB Flash Device selected
 
-#. When ready Press the  :guilabel:`Flash!` Button.
+#. When ready Press the :guilabel:`Flash!` Button.
 
-#. Flashing starting.
+#. The dialog shows :guilabel:`Flashing` while in progress.
```

Comment:
Steps 9-11 should not be steps IMO.  These are not actions for the user to take, so they shouldn't be number.  The should be just information for the previous step.

Reply:
I agree. Good suggestion.